cbrain_task_descriptors/oxford_asl.json

```diff
@@ -0,0 +1,454 @@
+{
+    "tool-version": "6.0.4",
+    "name": "oxford_asl",
+    "author": "Oxford Centre for Functional MRI of the Brain (FMRIB)",
+    "descriptor-url": "https://github.com/aces/cbrain-plugins-neuro/blob/master/cbrain_task_descriptors/oxford_asl.json",
+    "command-line": "oxford_asl [INPUT_FILE] [OUTPUT_DIR] [MASK] [SPATIAL] [WP] [MC] [IAF] [IBF] [TIS] [TIIMG] [CASL] [ARTSUPP] [BOLUS] [BAT] [T1] [T1B] [SLICEDT] [SLICEBAND] [FSLANAT] [S] [SBRAIN] [FASTSRC] [SENSCORR] [M0] [ALPHA] [C] [TR] [CMETHOD] [CBLIP] [PEDIR] [ECHOSPACING] [PVCORR]",
+    "schema-version": "0.5",
+    "container-image": {
+        "image": "mcin/docker-fsl:latest",
+        "index": "docker://",
+        "type": "singularity"
+    },
+    "inputs": [
+        {
+            "command-line-flag": "-i",
+            "description": "ASL data input",
+            "value-key": "[INPUT_FILE]",
+            "type": "File",
+            "list": false,
+            "optional": false,
+            "id": "asl_data",
+            "name": "Asl data"
+        },
+        {
+            "command-line-flag": "-o",
+            "description": "Output directory name",
+            "value-key": "[OUTPUT_DIR]",
```

Comment:
What checks are made on this? What prevents a user from entering "/home/myusername/.ssh" and trashing a system directory unrelated to the task?

Reply:
CBRAIN do some sanity check.